x-pack/plugins/security_solution/public/detection_engine/rule_creation_ui/components/query_bar/index.test.tsx

```diff
@@ -73,7 +73,6 @@ describe('QueryBarDefineRule', () => {
       <TestProviders>
         <Router history={mockHistory}>
           <QueryBarDefineRule
-            browserFields={{}}
```

Comment:
These look like changes from https://github.com/elastic/kibana/pull/187066; should they be here as well?

Reply:
hey yes, the pr's will be merged in order:) thanks!

Reply:
I was also confused at first, then looked at the list of commits to see your branched off of the first branch.
Maybe add a note in the PR description if others look it?